README.md

```diff
@@ -1,6 +1,6 @@
 # roblox-assets-grab
 
-More information about the archiving project can be found on the ArchiveTeam wiki: [Roblox assets](https://wiki.archiveteam.org/index.php?title=Roblox assets)
+More information about the archiving project can be found on the ArchiveTeam wiki: [Roblox assets](https://wiki.archiveteam.org/index.php/Roblox_assets)
```

Comment:
I suggest just linking to the Roblox page, as a section about this grab can be added there: https://wiki.archiveteam.org/index.php/Roblox